src/scripts/main.js

```diff
@@ -1,3 +1,45 @@
 'use strict';
 
-// write code here
+const persons = document.body.querySelectorAll('li');
+
+function sortSalary() {
+  let previous = persons[0]
+    .getAttribute('data-salary')
+    .replace((/[,$]/g, ''), 10);
+
+  for (let i = 1; i < persons.length; i++) {
+    const current = persons[i]
+      .getAttribute('data-salary')
+      .replace((/[,$]/g, ''), 10);
+
+    if (current > previous) {
+      persons[i - 1].before(persons[i]);
```

Comment:
The sorting logic is incorrect. Simply comparing `current` and `previous` and moving elements does not ensure a sorted list. Consider using a sorting algorithm or the `Array.prototype.sort` method for correct sorting.